### PR TITLE
Add chromosome_breakpoint SO term

### PR DESCRIPTION
**Pull request by nuno-agostinho**

### CODE REVIEW

File: modules/Bio/EnsEMBL/Variation/Utils/Constants.pm [hunk 1]
```diff
@@ -1,8 +1,9 @@
 package Bio::EnsEMBL::Variation::Utils::Constants;
 
-#####################################################################
-# NB: THIS FILE HAS BEEN AUTOMATICALLY GENERATED, EDIT WITH CAUTION #
-#####################################################################
+##############################################################################
+#   NB: AUTOMATICALLY GENERATED; AVOID MANUAL EDITING! EDIT Constants.pm     #
+#   AND THEN RUN perl scripts/misc/create_config_consts.pl > Config.pm       #
```
Comment (nakib103) [inline modules/Bio/EnsEMBL/Variation/Utils/Constants.pm:5]:
```suggestion
#   NB: AUTOMATICALLY GENERATED; AVOID MANUAL EDITING! EDIT Config.pm     #
#   AND THEN RUN perl scripts/misc/create_config_consts.pl > Constants.pm       #
```

File: scripts/misc/create_config_consts.pl [hunk 2]
```diff
@@ -86,9 +86,10 @@ =head1 CONTACT
 
 my $hdr = q{package Bio::EnsEMBL::Variation::Utils::Constants;
 
-#####################################################################
-# NB: THIS FILE HAS BEEN AUTOMATICALLY GENERATED, EDIT WITH CAUTION #
-#####################################################################
+##############################################################################
+#   NB: AUTOMATICALLY GENERATED; AVOID MANUAL EDITING! EDIT Constants.pm     #
```
Comment (nakib103) [inline scripts/misc/create_config_consts.pl:90]:
```suggestion
#   NB: AUTOMATICALLY GENERATED; AVOID MANUAL EDITING! EDIT Config.pm     #
```

File: scripts/misc/create_config_consts.pl [hunk 3]
```diff
@@ -86,9 +86,10 @@ =head1 CONTACT
...
+#   AND THEN RUN perl scripts/misc/create_config_consts.pl > Config.pm       #
```
Comment (nakib103) [inline scripts/misc/create_config_consts.pl:91]:
```suggestion
#   AND THEN RUN perl scripts/misc/create_config_consts.pl > Constants.pm       #
```

**Review by nakib103 — CHANGES_REQUESTED** [review]

**Comment by nuno-agostinho** [comment]
Sorry for the stupid issue 🙈 and thanks for the fix, @nakib103!

**Review by nakib103 — APPROVED** [review]
LGTM !~

**Comment by nakib103** [comment]
merged to `release/113` and `main`

**Pull request merged**